roc_example_test.go

```diff
@@ -77,7 +79,8 @@ func ExampleROC_equallySpacedCutoffs() {
 	n := 9
 
 	stat.SortWeightedLabeled(y, classes, weights)
-	cutoffs := stat.EquallySpaced(y[0], y[len(y)-1], n)
+	cutoffs := make([]float64, n)
+	floats.Span(cutoffs, math.Nextafter(y[0], y[0]-1), y[len(y)-1])
```

Comment:
One option would be to just panic if cutoffs is nil or empty, and make people construct cutoffs that would generate all possible values themselves. But I think its nice because that behaviour is often wanted.